modules/openapi-generator/src/main/resources/php/model_generic.mustache

```diff
@@ -371,7 +371,7 @@ class {{classname}} {{#parentSchema}}extends {{{parent}}}{{/parentSchema}}{{^par
      *
      * @return boolean
      */
-    public function offsetExists($offset)
+    public function offsetExists($offset): boolean
```

Comment:
```suggestion
    public function offsetExists($offset): bool
```

From the [PHP manual](https://www.php.net/manual/en/language.types.declarations.php):
> **Warning** Aliases for the above scalar types are not supported. Instead, they are treated as class or interface names. For example, using boolean as a type declaration will require the value to be an [instanceof](https://www.php.net/manual/en/language.operators.type.php) the class or interface boolean, rather than of type bool